### DISCUSSION TITLE
What's the difference between use lefthook or @evilmartians/lefthook

### DISCUSSION THREAD

**Discussion by OskarCali** [evilmartians/lefthook#822] [Q&A] [answered]
I'm developing a React website using Vite + Typescript + SWC, I use Biome for linting and format and want to use commitizen + commitlint for ensuring the commit names.

I want to use git hooks with lefthook, but I don't understand what package I need it or what's the difference between these.

Sorry if my English is not the best, it doesn't my natural language.

**Comment by mrexox** [comment, marked as answer]
Hey! Use `lefthook` package. The only difference is that `@evilmartians/lefthook` package contains all binaries while `lefthook` provides only the one suitable for your OS, so it weights less.

**Reply by OskarCali** [reply]
Perfect!! thank you so much.
